### ISSUE TITLE
add multi-output support for gradient boosting

### ISSUE THREAD

**Issue by mastoffel** [alan-turing-institute/autoemulate#57]
see e.g. https://scikit-learn.org/stable/modules/generated/sklearn.multioutput.MultiOutputRegressor.html
